### PR TITLE
Added shared utility helpers

### PR DESCRIPTION
**Pull request by aditya-matukumalli**
- Added `EResourceType`, `getResourceIdentifier`, `getSiblingIdentifier` and `isPackage` utility functions used across agreements and plugin-find-eresource.
- Needs to be added in before merging https://github.com/folio-org/ui-plugin-find-eresource/pull/1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change history for stripes-erm-components
 
+## 3.1.0 IN PROGRESS
```
Comment (doytch) [inline CHANGELOG.md:3]: I like to bump the version in package.json at this point. Allows you to depend on `^3.1.0` in whatever apps you use these features, and ensures you won't accidentally release a package that has this unreleased dependency.

File: index.js [hunk 2]
```diff
@@ -30,8 +30,11 @@ export { default as withKiwtFieldArray } from './lib/withKiwtFieldArray';
 
 // Functions, utilities, and misc.
 export { default as customPropertyTypes } from './lib/customPropertyTypes';
+export { default as EResourceType } from './lib/EResourceType';
```
Comment (doytch) [inline index.js:33]: Components go in the first block per file style.

**Comment by doytch** [comment]
It'd be a good idea to write tests for these utilities. These things that pull in data from various sources are tough to test outside of unit tests. Additionally, we're already planning on refactoring this stuff in the future so tests allow us to confidently refactor without being afraid that we're missing something.

File: index.js (hunk 2 above)
Comment (aditya-matukumalli) [reply index.js:33]: > It'd be a good idea to write tests for these utilities. These things that pull in data from various sources are tough to test outside of unit tests. Additionally, we're already planning on refactoring this stuff in the future so tests allow us to confidently refactor without being afraid that we're missing something.

Sounds good

Comment (aditya-matukumalli) [reply index.js]: Updated the PR with utility helpers tests.  Just FYI: `getResourceIdentifier` and ``getSiblingIdentifier` are currently being called in only one [context](https://github.com/folio-org/ui-agreements/blob/master/src/components/views/EResources.js#L231), whereas `EresourceType` is being called from 6 different contexts. All of them should be covered by these tests

**Pull request merged**